### DISCUSSION TITLE
Audio output glitching since updating to Sequoia

### DISCUSSION THREAD

**Discussion by patrickrushton** [ExistentialAudio/BlackHole#821] [General]
I've been using Blackhole quite happily until updating to Sequoia. Now the audio output glitches when I visit certain webpages in Safari, for example Amazon product pages. 

My setup is to have Blackhole together with my audio interface (IK Multimedia AXE I/O) on a multi-output device which is set as my default system output. Loading certain graphics/javascript intensive webpages in Safari causes the audio to glitch. If I change my default audio output back to my audio interface directly, then the glitches disappear.

I've experimented with drift control settings, but didn't seem to make a difference. Also, removing Blackhole from the multi-output device doesn't fix the glitches, so I think the problem might be how Sequoia now manages multi-output devices, but I'm curious if any other Blackhole users have come across the issue?

I'm using a MacBook Pro M2 Pro, 16GB ram.

**Comment by devinroth** [comment]
Definitely sounds like a multi-output problem. You could set BlackHole as the default output without a multi-output but you won’t be able to hear anything. Or use another application to forward the audio from BlackHole to your speakers.

Another option would be to try our paid app Dipper (https://dipper.audio) which eliminates the need for multi-outputs.
